### PR TITLE
Add some little comments

### PR DESCRIPTION
**Pull request by natacha-beck**

### CODE REVIEW

File: boutiquespaper.tex [hunk 1]
```diff
@@ -619,6 +622,7 @@ \subsubsection{CBRAIN}
     (\texttt{cbrain:inherits-from-class}) to the \boutiques descriptor
     to define the Ruby class that should be used as parent class for
     the application.
+  %% nb: Which tool use this ?
```
Comment (glatard) [inline boutiquespaper.tex]: Niak

File: boutiquespaper.tex [hunk 2]
```diff
@@ -638,6 +642,8 @@ \subsubsection{CBRAIN}
 through the file system, and it is also powerful as it enables the
 parallelization of complex workflows such as the FSL and Niak ones
 described later. 
+%% nb: The parralelization of FSL-randomise was removed in CBRAIN it 
+%% did not work properly, do we use the parralelization for other FSL tools
```
Comment (glatard) [inline boutiquespaper.tex]: we do for Melodic. Why didn't it work properly?

Comment (natacha-beck) [reply boutiquespaper.tex]: FSL-Randomize launch lot of really small job (less than a minute) and concatenate the result at the end. When we used the parallelization some jobs stay in the queue for a long time. We should have a mechanism that regroups the sub-job by batch before running it. As far as I remember.

**Comment by glatard** [comment]
Thanks for the comments. I'll close this PR as it now conflicts. I have integrated your comments about Singularity and FSL's paralelization.

**Pull request closed without merging**